src/auto-evo/simulation/niche/ChemosyntheticNiche.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+public class ChemosyntheticNiche : INiche
+{
+    private static readonly Compound Glucose = SimulationParameters.Instance.GetCompound("glucose");
+    private static readonly Compound ATP = SimulationParameters.Instance.GetCompound("atp");
```

Comment:
These should not be static variables, it's good enough to cache these per instance, static variables may cause issues by enforcing loading of the SimulationParameters instance too soon.